test/typescript/typings.ts

```diff
@@ -1,24 +1,25 @@
 // relative path uses package.json {"types":"types/index.d.ts", ...}
 
-import Aedes = require ('../..')
+import aedes = require ('../..')
+import { Client, AuthenticateError } from '../..'
```

Comment:
I think we should really expose the server as `aedes.Server`.

Basically we should add to `aedes.js`:

``` js
aedes.Server = aedes
```

In this way you can do:

```js
import { Server, Client, AuthenticateError } from 'aedes'
```

This should not be a semver-major change, but just a facility for those using ESM and Typescript.

Reply:
I would prefer the way I suggested, as it’s an additive change and it does not require a change in ts configuration.

Reply:
I might have not explained it clearly - it wouldn't require ts configuration change, but change in the way it's currently imported. In any case, I added the `Server` property as you suggested.